src/main/java/com/znsio/teswiz/runner/DeviceSetup.java

```diff
@@ -84,15 +85,64 @@ static void verifyAppExistsAtMentionedPath() {
         LOGGER.info(String.format("Update path to Apk: %s", appPath));
         if(appPath.equals(NOT_SET)) {
             appPath = getAppPathFromCapabilities();
+            String directoryPath = "src" + File.separator + "test" + File.separator + "resources" + File.separator + "sampleApps";
+            String fileName = appPath.split(File.separator)[appPath.split(File.separator).length - 1];
+            String filePath = directoryPath + File.separator + fileName;
+            if((new File(filePath)).exists()){
```

Comment:
This is a very complicated if-else-if-else logic.

Can we simplify it?

Reply:
simplified it, please check now